crates/holo_hash/src/hash.rs

```diff
@@ -160,9 +159,13 @@ impl<T: HashType> HoloHash<T> {
     ///
     /// For convenience, 36 bytes can also be passed in, in which case
     /// the location bytes will used as provided, not computed.
-    pub fn from_raw_32_and_type(mut hash: Vec<u8>, hash_type: T) -> Self {
+    pub fn from_raw_32_and_type(hash: Vec<u8>, hash_type: T) -> Self {
+        #[cfg(feature = "serialized-bytes")]
+        let mut hash = hash;
+
+        #[cfg(feature = "serialized-bytes")]
         if hash.len() == HOLO_HASH_CORE_LEN {
-            hash.append(&mut encode::holo_dht_location_bytes(&hash));
+            hash.append(&mut crate::encode::holo_dht_location_bytes(&hash));
         }
```

Comment:
You can probably put this entire method behind the feature flag. Anyone who uses it without the feature flag will need to pass in 36 bytes (32 bytes will cause a panicfrom the assert!), in which case they should just use `from_raw_36_*`.

Really I don't think we expect anyone to be using any of these `from_raw` methods in Wasm, but it's safe to do so (except for this one)